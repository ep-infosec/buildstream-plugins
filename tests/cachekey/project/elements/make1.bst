kind: make
sources:
- kind: local
  path: files/local
