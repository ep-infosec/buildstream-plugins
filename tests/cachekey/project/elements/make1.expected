9c297771102045d7619e678d952e5b0a8771d9f7afcb11e5bb072f9e84b8724c